tests/scripts/t3000/run_t3000_demo_tests.sh

```diff
@@ -31,8 +31,8 @@ run_t3000_falcon7b_tests(){
   WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest --disable-warnings -q -s --input-method=json --input-path='models/demos/t3000/falcon7b/input_data_t3000.json' models/demos/t3000/falcon7b/demo_t3000.py::test_demo_multichip[user_input0-8-True-default_mode_greedy_verify]
 
   # Falcon7B perplexity test (prefill and decode)
-  WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/falcon7b/tests/test_perplexity_falcon.py::test_perplexity[1-True-prefill_seq1024_dram]
-  # WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/falcon7b/tests/test_perplexity_falcon.py::test_perplexity[1-True-decode_1024_l1_sharded]  # Disabled due to Issue #9268
+  WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/falcon7b/tests/test_perplexity_falcon.py::test_perplexity[True-prefill_seq1024_dram]
+  WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/falcon7b/tests/test_perplexity_falcon.py::test_perplexity[True-decode_1024_l1_sharded]
```

Comment:
Noticing in the raw logs that there is a deprecation warning.
Should this be patched in this PR too?

Workflow link: https://github.com/tenstorrent/tt-metal/actions/runs/9519157120/job/26241907392

Raw Log link: https://productionresultssa11.blob.core.windows.net/actions-results/7e9c6ce2-3d4f-4394-b175-1fae85eaa3f4/workflow-job-run-b58f06d6-9baa-5eb3-9035-03009dd223b5/logs/job/job-logs.txt?rsct=text%2Fplain&se=2024-06-15T00%3A48%3A31Z&sig=vJxFRrchm9hR51HKSMS68ESeUJA8EylWFYTZvfD22U0%3D&ske=2024-06-15T09%3A34%3A35Z&skoid=ca7593d4-ee42-46cd-af88-8b886a2f84eb&sks=b&skt=2024-06-14T21%3A34%3A35Z&sktid=398a6654-997b-47e9-b12b-9515b896b4de&skv=2023-11-03&sp=r&spr=https&sr=b&st=2024-06-15T00%3A38%3A26Z&sv=2023-11-03
```
2024-06-14T17:01:39.2716048Z models/demos/falcon7b/tests/test_perplexity_falcon.py::test_perplexity[True-prefill_seq1024_dram]
2024-06-14T17:01:39.2717994Z   /home/ubuntu/actions-runner/_work/tt-metal/tt-metal/python_env/lib/python3.8/site-packages/huggingface_hub/file_download.py:1132: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
```